packages/kbn-rule-data-utils/src/alerts_as_data_rbac.ts

```diff
@@ -23,6 +23,7 @@ export const AlertConsumers = {
   SLO: 'slo',
   SIEM: 'siem',
   UPTIME: 'uptime',
+  THRESHOLD: 'threshold',
```

Comment:
Do we really want to add a new consumer for this?  From my brief understanding, feels like this should reuse `OBSERVABILITY`.  Unless it's intended to be super-generic or something.  In which case perhaps this should go in our eventual "stack" consumer (whatever we create for the stack alerting rules once they start generating alerts-as-data indices).

Reply:
@pmuellr, thanks for the feedback. I think we need to reuse `observability` as a consumer and not add a new one.